### ISSUE TITLE
[Analytics] Google Tag Manager

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#5935]
Topic: Google Tag Manager

- [x] Ben to work with Pomax to put the tag manager in place
- [x] Ben to give Taís access to tag manager
- [ ] Taís to do tutorials to catch up/use Tag Manager
- [x] Ben and Taís to discuss how to smoothly roll the updates to teammates - @kristinashu I feel you could be involved on this too, what do you think?
- [x] Ben and Taís to come up with a plan/process for Tag Manager transition: we can start tracking our events in tag manager post-implementation. A discussion that will have to happen is which events we should port to Tag Manager (probably the most important ones/prioritize to minimize effort) Other considerations: what happens to our existing dashboards, how do we set up new events, what people are interesting in tracking - @kristinashu I feel you could be involved on this too, what do you think?
- [ ] Onboard our teammates
- [ ] Document our transition plan and share with team

**Comment by jamilasnell** [comment]
@taisdesouzalessa estimating this as a 16 but please revise if need be!

**Comment by taisdesouzalessa** [comment]
Taís and Ben met to discuss those issues today and here are the decisions we made:
1. Transition plan to Tag Manager: GTM is on MozFest and Foundation site, not on Donate site due to its complexity. We can all use GTM from now on (thanks Ben!). We decided that only NEW EVENTS will be created on Tag Manager and old events will remain in the code. _I'll add this update to our Design/Engineer sync doc so we are all on the same page._

2. Onboard teammates: as Taís learn more about GTM she will write a documentation to onboard the other designers. Ben will review the documentation to make sure it makes sense. Once documentation is ready (and reviewed) we can do a brownbag session on it - for now, designers can also start looking at it/playing with GTM if they want (there is a review process in place so no event can be pushed live without an approval by Engineer)  _I'll add this update to our Design/Engineer sync doc so we are all on the same page._

Discussions are done, so I creates a new ticket for next steps: #6022

fyi: @mmmavis @sabrinang @kristinashu @natalieworth @Pomax @KalobTaulien @nancyt1
